src/main/webapp/app/components/sidebar/NavigationSidebar.tsx

```diff
@@ -232,7 +233,7 @@ export const NavigationSidebar: React.FunctionComponent<StoreProps> = props => {
           <Menu>
             {props.isCurator && (
               <MenuItemCollapsible
-                isCollapsed={props.isNavSidebarCollapsed}
+                isCollapsed={props.isNavSidebarCollapsed ?? false}
```

Comment:
For optional props within functional component, we should define a default instead of doing multiple checks in the same component.

Reply:
Yeah that makes sense. My mind wasn't thinking so clearly after hours of doing this. 😄